Shared/Samples/Render multilayer symbols/README.md

```diff
@@ -0,0 +1,42 @@
+# Render multilayer symbols
+
+Show different kinds of multilayer symbols on a map similar to some pre-defined 2D simple symbol styles.
+
+![Image of render multilayer symbols](render-multilayer-symbols.png)
+
+## Use case
+
+Allows you to customize a graphic with a multilayer symbol. For example, you may want more customizable symbols than the one that is provided with the API to display a unique representation of a landmark.
+
+## How to use the sample
+
+The sample loads multilayer symbols for points, polylines, and polygons.
+
+## How it works
+
+1. Create multilayer symbols for each predefined 2D simple symbol style.
+    * For multilayer point symbols, use `MultilayerPointSymbol(symbolLayers:referenceProperties:)`.
+    * For multilayer polyline symbols, use `MultiLayerSymbol(symbolLayers:referenceProperties:)`.
+    * For multilayer polygon symbols, use `MultiLayerPolygonSymbol(symbolLayers:referenceProperties:)`.
+2. Create graphics by passing in a geometry and the associated symbol.
+3. Add graphics to the graphics overlay with `graphicsOverlay.addGraphics(_ newGraphic:)`
```

Comment:
You can check how to correctly right out this method by doing this:
<img width="1231" alt="Screenshot 2023-09-25 at 9 13 19 AM" src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/74069582/9a9126bf-3bf2-47a5-83a8-489eff3e27e5">

```suggestion
3. Add graphics to the graphics overlay with `graphicsOverlay.addGraphics(_:)`
```